index.bs

```diff
@@ -575,68 +575,182 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		</div>
 
 		</div><!-- dfn-for Navigator -->
 
-	<h3 id="clipboard-interface">Clipboard Interface</h3>
-
+	<h3 id="clipboard-item-interface">ClipboardItem Interface</h3>
 		<pre class="idl" data-highlight="webidl">
-		typedef sequence&lt;ClipboardItem> ClipboardItems;
+			typedef (DOMString or Blob) ClipboardItemDataType;
+			typedef Promise&lt;ClipboardItemDataType&gt; ClipboardItemData;
 
-		[SecureContext, Exposed=Window] interface Clipboard : EventTarget {
-		  Promise&lt;ClipboardItems> read();
-		  Promise&lt;DOMString> readText();
-		  Promise&lt;undefined> write(ClipboardItems data);
-		  Promise&lt;undefined> writeText(DOMString data);
-		};
+			[SecureContext, Exposed=Window]
+			interface ClipboardItem {
+				constructor(record&lt;DOMString, ClipboardItemData> items,
+				            optional ClipboardItemOptions options = {});
 
-		typedef (DOMString or Blob) ClipboardItemDataType;
-		typedef Promise&lt;ClipboardItemDataType> ClipboardItemData;
+				readonly attribute PresentationStyle presentationStyle;
+				readonly attribute FrozenArray&lt;DOMString> types;
 
-		callback ClipboardItemDelayedCallback = ClipboardItemData ();
+				Promise&lt;Blob&gt; getType(DOMString type);
+			};
 
-		[Exposed=Window] interface ClipboardItem {
-		  constructor(record&lt;DOMString, ClipboardItemData> items,
-		    optional ClipboardItemOptions options = {});
-		  static ClipboardItem createDelayed(
-		      record&lt;DOMString, ClipboardItemDelayedCallback> items,
-		      optional ClipboardItemOptions options = {});
+			enum PresentationStyle { "unspecified", "inline", "attachment" };
 
-		  readonly attribute PresentationStyle presentationStyle;
-		  readonly attribute long long lastModified;
-		  readonly attribute boolean delayed;
+			dictionary ClipboardItemOptions {
+				PresentationStyle presentationStyle = "unspecified";
+			};
+		</pre>
 
-		  readonly attribute FrozenArray&lt;DOMString> types;
+		<p><dfn for="ClipboardItem">Clipboard Item</dfn></p>
 
-		  Promise&lt;Blob> getType(DOMString type);
-		};
+		A [=clipboard item=] is conceptually data that the user has expressed a desire to make shareable by invoking a "cut" or "copy" command.
+		For example, if a user copies a range of cells from a spreadsheet of a native application, it will result in one [=clipboard item=]. If a user copies a set of files from their desktop, that list of files will be represented by multiple [=clipboard item=]s.
+		Some platforms may support having more than one [=clipboard item=] at a time on the [=Clipboard=], while other platforms replace the previous [=clipboard item=] with the new one. 
+
+		A [=clipboard item=] has a <dfn>list of representations</dfn>, each <dfn>representation</dfn> with an associated <dfn>mime type</dfn> and <dfn>data</dfn>. In the example where the user copies a range of cells from a spreadsheet, it may be represented as an image (image/png), an HTML table (text/html), or plain text (text/plain).
```

Comment:
"With an associated mime type". It seems "mime type" should link to https://mimesniff.spec.whatwg.org/#mime-type. Unfortunately I'm not sufficiently familiar with ReSpec to write the markup for that.

Reply:
Ack.